src/content/docs/infrastructure/host-integrations/host-integrations-list/postfix-monitoring-integration.mdx

```diff
@@ -332,42 +314,40 @@ Flex comes bundled with the New Relic infrastructure agent. To create a flex con
                 set_header: [Bounce.msgCount, bounceMessage]
                 regex_match: true
                 split_by: \s*(\d+)\s\s\s(.*.)
-```
-
-  
+```        
+    </Step>
+    <Step>
 ## Forward your Postfix logs to New Relic
-  
-You can use our [log forwarding](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/) to forward Postfix logs to New Relic.
 
-On Linux machines, your log file named `logging.yml` should be present in this path:
+1. Create a file named `logging.yml` in the infrastrucutre agent directory: 
```

Comment:
same comment as sonarqube. what value do customers get by forwarding logs? how does opting out of log forwarding degrade their data?

Reply:
I'll add a quick intro sentence, but I don't think we should delve into explaining the big "whys" and "hows" of logs. I think it is outside the scope of an intro experience, and getting users an experience with as much data as possible is the priority.